### DISCUSSION TITLE
Trouble adding fonts

### DISCUSSION THREAD

**Discussion by Detroit755** [inkstitch/inkstitch#1571] [Q&A]
Hi…I’m trying to add fonts to the lettering tool in InkStitch and I keep getting this error when trying to generate json. I don’t know what it means or which step I’m doing incorrectly. It might be the step when I’m trying to save it and name it ➡️.svg it’s saving just as ➡️ Can someone tell me what i need to do…thanks
![Uploading 4743294F-44E6-4D25-B622-5493FCABBEC8.jpeg…]()

**Comment by tyrosinase** [comment]
This is probably better asked in Issues (https://github.com/inkstitch/inkstitch/issues). Discussions don't notify the devs and they miss things here a lot.

(So do I; if there's a way to get all the Discussions stuff mailed to you the way Issues does, I'd appreciated it if someone shared it with me)
